document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/mapping/smallfull/model/ixbrl/SmallFullAccountIxbrl.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.Gson;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.BalanceSheet;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.company.Company;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.notes.Notes;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.period.Period;
+
+import java.util.Objects;
+
+public class SmallFullAccountIxbrl {
+
+    @JsonProperty("period")
+    private Period period;
+
+    @JsonProperty("balance_sheet")
+    private BalanceSheet balanceSheet;
+
+    @JsonProperty("notes")
+    private Notes notes;
+
+    @JsonProperty("company")
+    private Company company;
+
+    @JsonProperty("approval_date")
+    private String approvalDate;
+
+    @JsonProperty("approval_name")
+    private String approvalName;
+
+    public Period getPeriod() {
+        return period;
+    }
+
+    public void setPeriod(Period period) {
+        this.period = period;
+    }
+
+    public BalanceSheet getBalanceSheet() {
+        return balanceSheet;
+    }
+
+    public void setBalanceSheet(BalanceSheet balanceSheet) {
+        this.balanceSheet = balanceSheet;
+    }
+
+    public Notes getNotes() {
+        return notes;
+    }
+
+    public void setNotes(Notes notes) {
+        this.notes = notes;
+    }
+
+    public Company getCompany() {
+        return company;
+    }
+
+    public void setCompany(Company company) {
+        this.company = company;
+    }
+
+    public String getApprovalDate() {
+        return approvalDate;
+    }
+
+    public void setApprovalDate(String approvalDate) {
+        this.approvalDate = approvalDate;
+    }
+
+    public String getApprovalName() {
+        return approvalName;
+    }
+
+    public void setApprovalName(String approvalName) {
+        this.approvalName = approvalName;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
could we add curly braces as per our coding standards?

Reply:
Auto generated hash and equals in IntelliJ for use in debugging.